src/App.jsx

```diff
@@ -7,24 +7,40 @@ import RestaurantList from "./components/Main/RestaurantList.jsx"
 import Restaurants from "./data/Restaurant.js";
 import { useState } from "react";
 
-function App() { 
+function App() {
   const [ category, setCategory ] = useState("전체");
   const restaurantList = Restaurants();
 
   const filteredRestaurants = category === "전체" ? restaurantList : restaurantList.filter(
     (restaurant) => restaurant.category === category
   );
 
+  const [ clickedRestaurantItem, setClickedRestaurantItem ] = useState(null);
+  const [ showDetailModal, setShowDetailModal ] = useState(false);
+
+  const handleOpenModal = (name, description) => {
+    const restaurant = {
+      name: name,
+      description: description
+    };
```

Comment:
```suggestion
  const handleOpenModal = (name, description) => {
    const restaurant = {
      name,
      description
    };
```

`key`와 `value`의 값이 같나요? 이렇게 축약할 수도 있답니다 😊

Reply:
와, 이 부분은 몰랐는데 알려주셔서 감사해요!!👍